### ISSUE TITLE
Glitches in the beginning of the exported videos if not editing with the frame-rate profile of the initial clip

### ISSUE THREAD

**Issue by Twenkid** [OpenShot/openshot-qt#4378]
**Describe the bug:**
For example a 30 fps video segment at the intro from other source (i.e. old 640x480x30 fps from Canon compact camera, either .avi or .mp4, then 50 fps from Canon Legria, final target 25 fps), however working with a profile of 50 fps. 

It is not the 30 fps though, because the same happens with 25 fps Legria mts sources with videos at 50 fps profile, also with other mp4 at 25 fps on a 50 fps profile, either raw or additionally encoded with ffmpeg commands, or downloaded from youtube.

Sometimes there is a desync of the framerate, it runs slowly, then skips frames, then continues. 

The issue is very nasty if one doesn't find the work-around, it has wasted me tons of time: it is visible only after exporting, and if one does not check the video afterwards (I always check due to expectation of glitches) and wasn't attentive at the beginning, she may end up uploading or presenting a video with nasty glitch right in the introduction... 

One workaround sometimes was to mask the first video segment with initial segments of titles or static pictures etc. but it's not always wanted and is additional work.

Recently I realized it might be something of the Profile of the edtor: (Ctrl+P, File->Profile)
It seemed this resolved it so far, but it shouldn't be like that or at least there should be a warning.

* Of the initial clip - I guess so; it reminds me of the design of an in-house editor of mine, which in order to save coding, instead of saving the format of the project explicitly in the project file, used the format of the first video clip as the project profile.

**Expected behavior:**

Smooth video playing as in the preview and no glitches.

**System Details:**
- OpenShot 2.5.1, at least since the beginning of that year (when first noticed)

**Screenshots:** (Optional)

You can see the behavior:

1) 640x480x30 fps (not NTSC, 30:1), profile 720/50 (but then exported to 640x480)
https://youtu.be/hjfdAIA3vlk  

A glitch, output is 30 fps, profile is 50: (also aspect, it is fixed by "stretch")
http://twenkid.com/video/30-profile-50fps.mp4

No glitch: profile is NTSC-640x480, output is the same:
http://twenkid.com/video/30-ntsc-profile-30-ntsc-export.mp4

2) Source: 720px25 mp4. At 50 fps it had а glitch. When switching to profile at 25 fps, it exported fine. When switching back to 50 fps, it again produced a glitch:
http://twenkid.com/video/glitch.mp4
Slow motion and it seems it takes sound from some broken buffer.

Another export and another glitch at the start with a frame taken somewhere from the clip (looks like 25-th frame):
http://twenkid.com/video/25.mp4

**Comment by speedytechdev** [comment]
Hi @Twenkid , Thank you for going into detail about the bug it helps our team a lot. Just to double-check the issue could you try and use our daily build. If the problem still occurs I would be happy to mark this as a bug.

**Comment by Twenkid** [comment]
A quick test for now, for the 640x480x30 fps it seemed to produce a smooth video once, but for another video it had the same bug. (I can't go deeper now). 

However I regret I tested it, because I see that daily build 2.6.0 had again another issue (due to which I didn't use the lastest version or daily builds): the preview with sound in the editor is at 2-3? fps for any video... :)) I prefered the previous bug, I found a way to fix it...

**Comment by speedytechdev** [comment]
Sorry about you having so many issues but we do really appreciate you bringing up the issues. Our team will hopefully be able to fix this bug soon for you. If you need anything else please feel free to reach out. 


Thank You and enjoy the rest of your day.
Speedytechlive - Support Team
